react/features/base/dialog/components/web/StatelessDialog.js

```diff
@@ -79,7 +78,7 @@ type Props = {
  * @type {ReactElement}
  */
 const ContextProvider = withContextFromProps({
-    i18n: PropTypes.object
+    i18n: Object
```

Comment:
I don't think this works as expected; at least it shows a warning for me locally. Seems like this should be a function that gets called that returns null or an Error. So something like
```
const ContextProvider = withContextFromProps({
    i18n: i18n => typeof i18n === 'object' ? null : new Error()
});
```
Essentially you'd do what prop-types is doing but you'd be doing it manually for this one instance.

Reply:
I don't see any such warning. Not with eslint nor flow. Where are you seeing the problem?

Reply:
I was seeing it at execution time when opening a modal.